News/Entities/Models/UserModel.cs

```diff
@@ -4,7 +4,7 @@ public class UserModel
 {
     public string? Id { get; set; }
 
-    public string? Username { get; set; }
+    public string? FirstName { get; set; }
```

Comment:
Will the model only transmit the first name?

Reply:
Yes now we dont have LastName in our user. We can add it if needed (also something like description about user and so on).